test/app_test.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mobile_test/app/app.dart';
+import 'package:mobile_test/home/home_screen.dart';
+
```

Comment:
suggestion:  I think widget tests are great.  I think having some more would be good especially for the custom widgets like timer_text and the expandable_fab.